### PR TITLE
Workaround large scale issue in LeafNodeCollada

### PR DESCRIPTION
**Pull request by jmirabel**
At the moment, when applying a large scale (e.g. unit conversion) to a LeafNodeCollada, one can get a very ugly result.

Related to https://github.com/stack-of-tasks/pinocchio/issues/1370

### CODE REVIEW

**Comment by jmirabel** [comment]
I'll wait for https://github.com/openscenegraph/OpenSceneGraph/issues/1020 before merging this.

File: src/urdf-parser.cpp [hunk 1]
```diff
@@ -223,6 +223,7 @@ namespace viewer {
         osgVector3 scale (1,1,1);
         toFloats (mesh.attribute("scale"), scale);
         meshNode->setScale(scale);
+        meshNode->applyScale();
```
Comment (jcarpent) [inline src/urdf-parser.cpp:226]: Why applyScale is not contained in setScale?

Comment (jmirabel) [reply src/urdf-parser.cpp:226]: `applyScale` modifies the underlying geometry to apply the scale and reset the variable scale to 1. For instance:
```python
node->setScale(osgVector3(0.1, 0.1, 10));
node->applyScale();
scale = node->getScale(); // contains (1., 1., 1.)
```
Here the geometry inside the node has been modified (the points of your geometry).
While this
```python
node->setScale(osgVector3(0.1, 0.1, 10));
scale = node->getScale(); // contains (0.1, 0.1, 10)
```
does not modify the geometry inside.

The name `applyScale` may be misleading. In all cases, the scale is applied. Maybe `applyScaleToGeometry` or `setGeometryScale` is better ?

File: src/node-property.cpp [hunk 2]
```diff
@@ -155,7 +155,7 @@ namespace viewer {
         "  gui.callVoidProperty(nodeName,\"%1\")");
     QPushButton* button = new QPushButton(objectName());
     button->setToolTip (toolTip.arg(objectName()));
-    connect(button, SIGNAL(clicked()), SLOT(set(void)));
```
Comment (jcarpent) [inline src/node-property.cpp:158]: You are using a C++11 feature.

Comment (jmirabel) [reply src/node-property.cpp:158]: Yes, and this time, it is on purpose. I will make it clear in CMake.

**Comment by jmirabel** [comment]
I'll merge this since the aforementioned issue seems frozen.

**Pull request merged**